src/MetricExporter.ts

```diff
@@ -70,7 +70,7 @@ export class DynatraceMetricExporter implements PushMetricExporter {
 		};
 
 		// eslint-disable-next-line deprecation/deprecation
-		const apiToken = config.apiToken ?? config.APIToken;
+		const apiToken = config.apiToken ?? config.APIToken; // NOSONAR
```

Comment:
why does this need to be ignored by sonar? :thinking:

Reply:
Because the APIToken is deprecated inf favor of apiToken (and this line is just to make sure if you use the old spelling it still works)